src/fontra/views/applicationsettings/applicationsettings.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link href="/css/core.css" rel="stylesheet" />
+    <link href="/css/tooltip.css" rel="stylesheet" />
+    <title>Fontra Application Settings</title>
+    <script type="module" src="/web-components/modal-dialog.js"></script>
+    <style>
+      body {
+        overflow: scroll;
+      }
+      .main-container {
+        display: grid;
+        grid-template-columns: 13em auto;
+        padding: 0em;
+        box-sizing: border-box;
+        gap: 0em;
+      }
+
+      #header-container {
+        display: grid;
+        align-content: start;
+        gap: 1em;
+        padding: 2em 1em 2em 2em;
+      }
+
+      .header {
+        cursor: pointer;
+        font-size: 1.25em;
+        font-weight: bold;
+        text-underline-offset: 0.15em;
+      }
+
+      .header:hover {
+        text-decoration: underline dotted;
+      }
+
+      .header.selected {
+        text-decoration: underline;
+      }
+
+      .application-settings-panel {
+        display: grid;
+        padding: 2em 2em 2em 1em;
+        outline: none;
+      }
+
+      .application-settings-panel[hidden] {
+        display: none;
+      }
+
+      #panel-container {
+        /* overflow: scroll; */
+      }
+    </style>
+  </head>
+  <body>
+    <modal-dialog></modal-dialog>
+    <div class="main-container">
+      <div id="header-container"></div>
+      <div id="panel-container"></div>
+    </div>
+  </body>
+  <script type="module">
+    import { ApplicationSettingsController } from "/applicationsettings/applicationsettings.js";
+
+    async function startApp() {
+      window.applicationSettingsController =
```

Comment:
`start()` doesn't return anything, so this statement should be done in two parts:
- assign `applicationSettingsController`
- call `start()` on `window.applicationSettingsController()`

Reply:
I changed it, but I am not 100% sure if it's right – why I keep this feedback open.

Reply:
You got the await wrong. The constructor is (by definition) not async, but `start()` is (as per `async` keyword.

Reply:
sorry and thanks. done.